### PR TITLE
Feature/export csv

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <testsuites>
         <testsuite name="Project Test Suite">
             <directory>src/*/*/*/*Bundle/Tests</directory>
+            <directory>src/*/*/*Bundle/Tests</directory>
```
Comment (eddiejaoude) [inline phpunit.xml]: Do you need both of these?

Comment (brettminnie) [inline phpunit.xml]: currently yes, I need to renormalise the other bundle name

On Thu, Oct 15, 2015 at 1:17 PM, Eddie Abou-Jaoude <notifications@github.com

File: src/TransformCore/PHE/ExporterBundle/Model/Record.php [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    }
+
+    /**
+     * @param Hindrances $hindrances
+     * @return array
+     */
+    protected function extractHindrances(Hindrances $hindrances)
+    {
+        return array(
+            "hindrancesNoTime"       => (true === $hindrances->isNoTime()) ? "true" : "false",
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Record.php]: true/false as strings?

Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Model/Record.php]: to be used elsewhere, if you export booleans they don't cast to a string, else it's an integer, true or false is more descriptive in the context

Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Record.php]: ok

File: phpunit.xml (hunk 1 above)
Comment (eddiejaoude) [inline phpunit.xml]: ok

**Pull request merged**